src/site/_includes/layouts/markdown-content-page.njk

```diff
@@ -0,0 +1,25 @@
+{%- extends "layouts/generic.njk" -%}
+
+{%- from "components/side-nav/macro.njk" import guidanceSideNav -%}
+
+{%- block body -%}
+
+<div class="guidance-split-pane govuk-width-container">
+  <div class="guidance-split-pane__nav">
+    {{ guidanceSideNav({
+      url: page.url,
+      items: collections[tags[0]] | sort(false, false, 'data.order')
+    }) }}
+  </div>
+  <div class="guidance-split-pane__content">
+    <main class="guidance-content" id="main-content" role="main">
+      <h1 class="govuk-heading-xl">{{ title }}</h2>
+
+      <div class="govuk-body">
+        {{ content | safe }}
```

Comment:
Never nice to see safe, but it's needed here – see https://www.11ty.dev/docs/layouts/